api/src/main/java/org/apache/iceberg/DataFile.java

```diff
@@ -59,8 +59,9 @@ static StructType getType(StructType partitionType) {
         optional(128, "upper_bounds", MapType.ofRequired(129, 130,
             IntegerType.get(), BinaryType.get())),
         optional(131, "key_metadata", BinaryType.get()),
-        optional(132, "split_offsets", ListType.ofRequired(133, LongType.get()))
-        // NEXT ID TO ASSIGN: 134
+        optional(132, "split_offsets", ListType.ofRequired(133, LongType.get())),
+        optional(134, "sequential_number", LongType.get())
```

Comment:
I think the field should be named "sequence_number".

Reply:
agreed